ember-client/shared/src/main/scala/org/http4s/ember/client/EmberClientBuilder.scala

```diff
@@ -309,7 +311,7 @@ final class EmberClientBuilder[F[_]: Async: Network] private (
           address: UnixSocketAddress,
       ): Resource[F, Response[F]] =
         Resource
-          .eval(ApplicativeThrow[F].catchNonFatal(unixSockets.getOrElse(UnixSockets.forAsync[F])))
+          .eval(ApplicativeThrow[F].catchNonFatal(unixSockets.orElse(defaultUnixSockets).get))
```

Comment:
Should this use a more descriptive error?

Reply:
Good point, done :)